src/modules/vtol_att_control/tiltrotor.cpp

```diff
@@ -100,6 +106,13 @@ Tiltrotor::parameters_update()
 	param_get(_params_handles_tiltrotor.elevons_mc_lock, &l);
 	_params_tiltrotor.elevons_mc_lock = l;
 
+	/* avoid parameters which will lead to zero division in the transition code */
+	_params.front_trans_dur = math::constrain(_params.front_trans_dur, 0.5f, 10.0f);
```

Comment:
I started out trying something like this too but decided I was just cheating. I was thinking what this approach might bring if we standardise on it. Do we run the risk of having weird "hiccups" when things like this actually do run for 0.5 secs? It felt cleaner to allow for 0 to define "off" and then really skip any code that would otherwise run.

What about doing a one time check and setting something like ```arsp_blending_active = true|false```?

I can see you are trying to simplify the logic in the code elsewhere which makes since, just not sure if this approach is "correct".